docs/fides/docs/development/developing_fides.md

```diff
@@ -0,0 +1,216 @@
+
+### Docker
+
+Given that the majority of commands rely on Docker images, the first step is going to be getting some Docker images built. This is handled by the `build` session. As an excercise, let's check the docs for the `build` session via the `usage` session:
+
+```sh
+nox -s usage -- build
+```
+
+The simplest way to build everything is to run `nox -s build`. This will build all available Docker images used for development and testing. Additionally, to build only a specific image(s), run `nox -s usage -- build` to see what params have been documented.
+
+### Running the Webserver
+
+Now that all of the potential images we'll need are built, it's time to spin up the webserver:
+
+`nox -s dev`
+
+Depending on your computer, this will take anywhere from ~30 seconds to ~2 minutes. The webserver will log all of its activity to that window which is useful for debugging and checking potential server-side errors.
+
+!!! warning
+    When running `nox -s dev`, if you get an `importlib.metadata.PackageNotFoundError: fides`, it means that `fides` was not properly installed. Run `nox -s dev -- shell`, and then run `pip install -e .`
+
+    You can verify Fides is installed with `pip list | grep fides`
+
+### Opening a Shell
+
+Now that the webserver is up and running, a convenient way to interact with it as well as the Fides CLI is to open a shell directly into the running webserver container. Open a new terminal and run the following:
+
+`nox -s shell`
+
+With the shell open, we'll need to authenticate before we can start using the CLI. Run the following to authenticate the shell and get ready to run the CLI:
+
+`fides user login`
+
+With that done, we can do things like evaluations:
+
+`fides evaluate`
+
+Listing all resources of a type:
+
+`fides ls system`
+
+and more. Run `fides -h` to see the full list of commands offered by the CLI.
+
+## Checking Your Changes
+
+Once you've made some changes, it's time to verify that they are working as expected.
+
+If you've made changes to the CLI, you can verify them directly using the `shell` method described [here](#opening-a-shell).
+
+If you've made an API change, there are a few different ways to verify your changes. The first and most simple is to go to `localhost:8080/docs` and verify that the changes you made show up as expected in the documentation. This is most useful when adding a new endpoint or changing a schema.
+
+To dig in even deeper, you can use the **Authorize** button at the top of the API docs page to login (credentials are available in the `.fides/fides.toml` config file). This will allow you to send requests, view responses, etc.
+
+## Running Tests
+
+To keep things organized and to prevent this page from getting overloaded, the documentation for testing has its own page. Head over to [Testing](testing.md) for a full walkthrough of testing Fides and running test environments.
```

Comment:
I'd just say `Head over to [Testing](testing.md) for a full walkthrough of testing Fides and running test environments.`  and drop the part about why it's not on this page